central/notifications/datastore/internal/writer/writer_impl.go

```diff
@@ -0,0 +1,139 @@
+package writer
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/notifications/datastore/internal/store"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/uuid"
+)
+
+var (
+	_ Writer = (*writerImpl)(nil)
+
+	rootNamespaceUUID = uuid.Nil
```

Comment:
Wonder how future proof this is, for other occurences we either a) use `NewV5FromNonUUIDs` and a static string or b) a static UUID.

Let's just generate a static UUID and use it as the namespace here, to avoid clashes in case anyone else thinks about using the `Nil` UUID as namespace.